Remove dependency on .NET include file

* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`) 5.0.0-alpha.1.19564.1
* Windows version: (`winver`) 1909 (OS Build 10.0.18363.476)
* Does the bug reproduce also in WPF for .NET Framework 4.8?: No
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? No

 **Problem description:**  Build failure when suitable .NET SDK is not installed.
 
**Repro steps:**
1. New machine with Win10 1909 (hence .NET 4.8).
2. Setup machine per [WPF developer guide](https://github.com/dotnet/wpf/blob/master/Documentation/developer-guide.md).
3. Fork/clone master branch of dotnet/wpf.
4. build 

 **Actual behavior:** 
D:\repos\wpf\src\Microsoft.DotNet.Wpf\src\DirectWriteForwarder\CPP\DWriteWrapper\Common.h(15,10): fatal error C1083: Cannot open include file: 'CorError.h': No such file or directory [D:\repos\wpf\src\Microsoft.DotNet.Wpf\src\DirectWriteForwarder\DirectWriteForwarder.vcxproj]
 
 **Expected behavior:** No build failure.
 
This failure goes away after installing .NET 4.6.1 SDK, which makes a version of CorError.h visible to 
DirectWriteForwarder.vcxproj.

--- a/src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/CPP/DWriteWrapper/Common.h
+++ b/src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/CPP/DWriteWrapper/Common.h
@@ -12,7 +12,19 @@ using namespace System::Security;
 namespace MS { namespace Internal { namespace Text { namespace TextInterface { namespace Native
 {    
 #include "DWrite.h"
-#include "CorError.h"
+
+// --Begin-- Copied from CorError.h, 
+// which is not visible to WPF .NetCore builds without installing .NET 4.6.1 SDK
+#ifndef FACILITY_URT
+#define FACILITY_URT            0x13
+#endif
+#ifndef EMAKEHR
+#define SMAKEHR(val) MAKE_HRESULT(SEVERITY_SUCCESS, FACILITY_URT, val)
+#define EMAKEHR(val) MAKE_HRESULT(SEVERITY_ERROR, FACILITY_URT, val)
+#endif
+
+#define COR_E_INVALIDOPERATION EMAKEHR(0x1509)
+// --End-- copy from CorError.h
 
 private ref class Util sealed
 {

